addons/project/views/project_project_views.xml

```diff
@@ -458,7 +458,12 @@
                                                         <span style="font-weight:bold;" class="fa mt4 fa-meh-o text-warning" t-elif="record.rating_avg.raw_value &gt;= 2.33" title="Average Rating: Okay" role="img" aria-label="Neutral face"/>
                                                         <span style="font-weight:bold;" class="fa mt4 fa-frown-o text-danger" t-else="" title="Average Rating: Dissatisfied" role="img" aria-label="Sad face"/>
                                                     </b>
-                                                    <field name="rating_avg_percentage" widget="percentage"/>
+                                                    <t t-if="record.rating_avg.raw_value % 1 == 0">
```

Comment:
Same here

Reply:
> Same here

I'm not getting your point.
you want something like his.because the xml side doesn't work.
`int(record.rating_avg.raw_value) == 0 if record.rating_avg.raw_value.is_integer() else round(record.rating_avg.raw_value, 1) == 0 `

Reply:
ah, yes, sorry, I think I went a bit too fast, you can disregard my comment :slightly_smiling_face: